packages/fxa-auth-server/lib/routes/subscriptions.js

```diff
@@ -1408,9 +1399,6 @@ const directRoutes = (
         response: {
           schema: validators.subscriptionsStripeIntentValidator,
         },
-        validate: {
-          payload: {},
```

Comment:
Removed this payload validator, since this call doesn't actually take any params. Otherwise, had to send an empty `{}` in the request.